src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -167,10 +171,19 @@ function getObjectDisplayValue(element: SettingsTreeSettingElement): IObjectData
 				data: data[key],
 				options: additionalValueEnums,
 			},
+			removable: true,
 		});
 	});
 
-	return items;
+	return items.sort((a, b) => {
+		if (a.key.data < b.key.data) {
```

Comment:
Still not crazy about sorting the user's keys, but it doesn't have to be fixed now.